etc/kayobe/kolla/config/ironic/ironic-conductor.conf

```diff
@@ -0,0 +1,92 @@
+[DEFAULT]
+# Make direct deploy faster, transfer sparse qcow2 images
+force_raw_images = False
+# Avoid some rpc timeouts
+rpc_response_timeout = 360
+
+[agent]
+# Use iPXE server for direct deploy source
+image_download_source = http
+# Reduce load by only collecting logs on failure
+deploy_logs_collect = on_failure
+
+[console]
+socat_address = {% raw %}{{ api_interface_address }}{% endraw %}
+
+[conductor]
+automated_clean=true
+# We have busy conductors failing to heartbeat
+# Default is 10 secs
+heartbeat_interval = 30
+# Default is 60 seconds
+heartbeat_timeout = 360
+sync_local_state_interval = 360
+
+[agent]
+# Use iPXE server for direct deploy source
+image_download_source = http
+# Reduce load by only collecting logs on failure
+deploy_logs_collect = on_failure
+
+[console]
+socat_address = {% raw %}{{ api_interface_address }}{% endraw %}
+
+[conductor]
+automated_clean=true
```

Comment:
I saw at some sites we have this cleaning tweak from the Ironic docs:

```
[deploy]/erase_devices_priority=0
[deploy]/erase_devices_metadata_priority=0
[conductor]/clean_step_priority_override=deploy.erase_devices_express:5
```

I know cleaning is very site specific. Probably safer to leave the above out and default to scrub. Perhaps worth linking to the Ironic Doc?

Reply:
I would actually like our default to be fail if secure erase fails, with a note on how to work around that.

Reply:
I explicitly set a value for erase_devices as the priroity is dependent on the hardware manager in use. What do you think?